src/components/common/Dialogs/CreateDomainDialog/CreateDomainDialog.tsx

```diff
@@ -74,26 +75,22 @@ const CreateDomainDialog = ({
           domainColor: DomainColor.LightPink,
           domainPurpose: '',
           annotationMessage: '',
-          // motionDomainId: Id.RootDomain,
+          motionDomainId: Id.RootDomain,
         }}
         actionType={actionType}
         validationSchema={validationSchema}
         transform={transform}
         onSuccess={close}
       >
-        {({ watch }) => {
-          const forceActionValue = watch('forceAction');
-          if (forceActionValue !== isForce) {
-            setIsForce(forceActionValue);
-          }
-          return (
-            <CreateDomainDialogForm
-              back={prevStep && callStep ? () => callStep(prevStep) : undefined}
-              colony={colony}
-              enabledExtensionData={enabledExtensionData}
-            />
-          );
-        }}
+        {() => (
+          <CreateDomainDialogForm
+            back={prevStep && callStep ? () => callStep(prevStep) : undefined}
+            colony={colony}
```

Comment:
We don't really need to pass colony down various level, it can be accessed via context where its consumed

Reply:
E.g inside `useActionDialogStatus` and `DialogHeading`

Reply:
These components are inside a `Dialog`, so they shouldn't be able to access the colony context